packages/x-date-pickers/src/tests/describeValue/testPickerOpenCloseLifeCycle.tsx

```diff
@@ -245,11 +283,21 @@ export const testPickerOpenCloseLifeCycle: DescribeValueTestSuite<any, 'new-pick
       const newValue = setNewValue(values[0], { isOpened: true });
 
       // Dismiss the picker
-      userEvent.mousePress(document.body);
+      if (pickerParams.type === 'date-range') {
+        // for range picker a mouse press does not seem to trigger onClose (at least in "jsdom" env)
```

Comment:
If pressing `Escape` works for all pickers, I'm fine just replacing.
The way we close the picker is an implementation detail here anyway.

If no solution works for all, then I'm fine with this `if`

Reply:
The issue is that we have two separate tests, one asserting that `Esc` click closes and the other asserting that click outside closes pickers. 🤔 
Identifying why range picker does not seem to close in test suite might be worthwhile. 🤔

Reply:
Oh sorry I did not check what the test was actually doing
If the range pickers does not work with click outside, I would bail out with an early return and add a todo to check why later.

Reply:
I struggled on similar problems in the past :grimacing: